src/rdb/rdb_raft.c

```diff
@@ -734,7 +734,7 @@ rdb_raft_unpack_chunk(daos_handle_t slc, d_iov_t *kds, d_iov_t *data, int index)
 	unpack_arg.eph = index;
 	unpack_arg.slc = slc;
 
-	return dss_enum_unpack(invalid_oid, (daos_key_desc_t *)kds,
+	return dss_enum_unpack(invalid_oid, (daos_key_desc_t *)kds->iov_buf,
```

Comment:
Can we also remove the daos_key_desc_t casting? I think it's unnecessary and the root cause behind the bug.

Reply:
ok